kreuzberg/_ocr/_easyocr.py

```diff
@@ -180,7 +181,7 @@ async def process_image(self, image: Image.Image, **kwargs: Unpack[EasyOCRConfig
         try:
             result = await run_sync(
                 self._reader.readtext,
-                image.tobytes(),
+                np.array(image),
```

Comment:
I see, please import numpy locally inside the function - reason is, its a dependency of easyocr but not Kreuzberg.